repos/fdbt-site/tests/pages/api/typeOfCap.test.ts

```diff
@@ -58,9 +58,26 @@ describe('ticketRepresentation', () => {
             Location: '/selectCappedProductGroup',
         });
 
-        expect(updateSessionAttributeSpy).toBeCalledTimes(0);
+        expect(updateSessionAttributeSpy).toBeCalledTimes(1);
     });
-    it('should return 302 redirect to /  defineCapPricingPerDistance when type of cap is byDistance', () => {
+
+    it('should return 302 redirect to /multiTapsPricing when type of cap is byDistance', () => {
+        const { req, res } = getMockRequestAndResponse({
+            body: {
+                typeOfCap: 'byTaps',
+            },
+            mockWriteHeadFn: writeHeadMock,
+        });
+        typeOfCap(req, res);
+
+        expect(writeHeadMock).toBeCalledWith(302, {
+            Location: '/multiTapsPricing',
+        });
+
+        expect(updateSessionAttributeSpy).toBeCalledTimes(1);
```

Comment:
We shouldnt be checking the number of times its being called, we should be checking the content.

Reply:
it was written earlier, hence updated and will check the content then